src/lib/coaler/embedder/ConformerEmbedder.cpp

```diff
@@ -5,15 +5,47 @@
 #include "ConformerEmbedder.hpp"
 
 #include <GraphMol/DistGeomHelpers/Embedder.h>
+#include <GraphMol/FMCS/FMCS.h>
 #include <GraphMol/ForceFieldHelpers/UFF/UFF.h>
 #include <GraphMol/MolAlign/AlignMolecules.h>
 #include <GraphMol/SmilesParse/SmartsWrite.h>
+#include <GraphMol/SmilesParse/SmilesWrite.h>
 #include <GraphMol/Substruct/SubstructMatch.h>
 #include <spdlog/spdlog.h>
 
+#include <utility>
+
+#include "Forward.hpp"
+
 const unsigned seed = 42;
 const float forceTol = 0.0135;
 
+namespace {
+    RDKit::SubstructMatchParameters get_substructure_match_params_for_optimizer_generation() {
```

Comment:
CamelCase?

Reply:
anonymous namespace helper methods are named in snake case in the AMD codestyle, thought id stick to that :)